.github/workflows/build.yml

```diff
@@ -34,11 +34,29 @@ jobs:
       with:
         path: |
           output/**/*.conda
+
+    - name: Get JWT
+      id: get_token
+      if: github.event_name == 'push'  # do not upload on PR
+      uses: actions/github-script@v6
```

Comment:
i would suggest pinning this by hash and adding a comment for the version for stability as in https://github.com/freundTech/typst-forge/blob/13ee793295365281bdb95c858a68dc3eca0b7b94/.github/workflows/pre-commit.yml#L16

dependabot can update the comment